hedera-node/hedera-app/src/test/java/com/hedera/node/app/HederaVirtualMapStateTest.java

```diff
@@ -29,7 +26,7 @@ class HederaVirtualMapStateTest extends MerkleTestBase {
      */
     @BeforeEach
     void setUp() {
-        virtualMapState = new HederaVirtualMapState(CONFIGURATION, new NoOpMetrics());
+        virtualMapState = new HederaVirtualMapState(CONFIGURATION, new NoOpMetrics(), Time.getCurrent());
```

Comment:
The original code was initialized with `FakeTime`. Is using `Time.getCurrent()` correct?

Reply:
Currently, both options work and seem to be correct. But anyway, I updated `Time.getCurrent()` to `FakeTime` as it appears to be more contextually correct.